pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -570,7 +569,7 @@ func pickOneNodeForPreemption(nodesToVictims map[string]*extenderv1.Victims) str
 		// Get earliest start time of all pods on the current node.
 		earliestStartTimeOnNode := util.GetEarliestPodStartTime(nodesToVictims[node])
 		if earliestStartTimeOnNode == nil {
-			klog.ErrorS(errors.New("earliestStartTime is nil for node"), "should not reach here", "node", node)
+			klog.ErrorS(nil, "EarliestStartTime is nil for node. Should not reach here.", "node", node)
```

Comment:
Just wondering what's these kind of change benefit us?

Reply:
Yeah. I understand and I fixed them all, thank you very much.